public/docs/ts/latest/cookbook/production-deploy.jade

```diff
@@ -0,0 +1,327 @@
+
+  For one, it has a lot of files that aren't even used when serving your site to users since it has
+  all the development only packages. 
+  You could cut down the number of files by doing a production install (`npm install --production`),
+  but only *after* compiling your Typescript files.
+
+  It also has a lot of small files. Each of those is going to take a whole round trip to get
+  delivered to the browser, so it's not going to be that fast to load.
+
+  There's more you can do to make our site fast for users.
+ 
+.l-main-section
+:marked
+  ## Minimizing your payload
+
+  One of the easiest optimizations you can do is to reduce the code users have to download.
+  
+  There are several techniques for achieving a smaller payload for your app:
+
+  - Bundling: concatenating several modules into a single file (bundle). 
+  - Inlining: adding html and css as strings inside components.
+  - Minification: removing all unnecessary whitespace and optional tokens.
+  - Uglification: rewriting code to use shorter variable/function names.
+  - Dead Code Elimination: removing code/variables that are not used.
+  - Tree shaking: removing unused module exports.
+  - Ahead-of-Time (AoT) Compilation: pre-compiling Angular templates.
+
+  Each of these do different things overall, but they all work together and their effects
+  compound on each other.
+
+  ### Fewer files
+  Bundling and inlining reduce your app's load time by serving fewer files and thus
+  fewer round trips. 
+  For small files, the overwhelming majority of time it takes to get it is not spent downloading,
+  but rather communicating with the server and coordinating the transfer. 
+  You can read more about resource timings in the 
+  [Chrome DevTools Network Performance page](https://developers.google.com/web/tools/chrome-devtools/network-performance/understanding-resource-timing)   
+
+  ### Smaller files
+  Minification, Uglification and Dead Code Elimination all contribute to reduce the total size
+  of your code to the smallest possible amount of bytes, which results in a smaller transfer time.
+
+  Tree shaking also reduces the total size but does so by removing whole exports from modules.
+  It was popularized by [Rollup](http://rollupjs.org/) and you can read more about it in 
+  [this post](https://medium.com/@Rich_Harris/tree-shaking-versus-dead-code-elimination-d3765df85c80#.15ih9cyvl).
+
+  ### Template compilation
+  AoT compilation reduces render time by pre-compiling Angular templates that otherwise would be
+  compiled at runtime. 
+  It also reduces payload size since the compiled app does not depend on the Angular compiler,
+  nor on declared directives that are never used in templates. 
+  This allows Tree-Shaking to further remove exports.
+
+  You can read more about AoT Compilation in our [dedicated cookbook chapter](aot-compiler.html),
+  where you'll also find instructions on how to use Rollup to perform all the optimizations shown here.
```

Comment:
I suggest removing the word `Rollup` from here. Why? Because we may use something else later and it would be easier to edit just that doc later and not this one too.